rust/sbp/src/serialize.rs

```diff
@@ -105,13 +107,13 @@ impl SbpSerialize for f64 {
     }
 }
 
-impl SbpSerialize for String {
+impl SbpSerialize for SbpString {
     fn append_to_sbp_buffer(&self, buf: &mut Vec<u8>) {
-        buf.extend(self.as_bytes());
+        buf.extend(self.0.clone());
```

Comment:
Could we add a `as_bytes()` to `SbpString` as well? I think cloning here is going to make another copy that we could avoid.